server/index.js

```diff
@@ -33,7 +34,9 @@ app
     .use(slashes());
 // TODO: csrf, gzip
 
+
 app.use(router);
+app.use(errorHandler);
```

Comment:
it should be moved to line 48, otherwise next part piece of code won't work as expected.